src/audio-worklet/migration/spn-recorder/app.js

```diff
@@ -215,8 +223,9 @@ function setupVisualizers() {
 
   function draw() {
     if (currentSamples) {
-      // Calculate current sample's average gain for visualizers to draw with.
-      // We only need to calculate this value once per render frame.
+      // Calculate current sample's average gain for visualizers to 
```

Comment:
Perhaps:

Calculate the average gain of collected samples for the visualization. This needs to be done once per frame.